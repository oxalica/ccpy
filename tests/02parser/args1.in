def f(a, b=2, c=3, *d):
  return a, b, c, d

def g(a=None):
  return a

def h(*x):
  return x
