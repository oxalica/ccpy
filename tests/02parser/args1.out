StmtDef("f", [("a", None), ("b", Some(ExprLiteral(LitInteger(2)))), ("c", Some(ExprLiteral(LitInteger(3))))], Some("d"), [StmtReturn(Some(ExprTuple(ExprName("a"), ExprName("b"), ExprName("c"), ExprName("d"))))])
StmtDef("g", [("a", Some(ExprLiteral(LitNone)))], None, [StmtReturn(Some(ExprName("a")))])
StmtDef("h", [], Some("x"), [StmtReturn(Some(ExprName("x")))])
End
